docs/framework/deployment/index.md

```diff
@@ -39,9 +39,9 @@ This article helps you get started deploying the .NET Framework with your applic
   
     -   Installer options:  
   
-        -   [Web installer](../../../docs/framework/getting-started/install-the-net-framework.md#choices)  
+        -   [Web installer](~/docs/framework/install/guide-for-developers.md#installation-choices)  
```

Comment:
For both this and the next link, the link is to an anchor tag that's too high in the document -- we end up linking to a section on developer packs, which is irrelevant for deployment. We want to link to the section #to-install-or-download-the-.net-framework-redistributable.